src/app/shared/language-selector/language-selector.component.html

```diff
@@ -1,6 +1,6 @@
 <mat-form-field id="language-selector">
   <mat-label>Language</mat-label>
-  <mat-select [formControl]="languageSelector">
+  <mat-select [formControl]="languageSelector" (selectionChange)="setLanguage()" class="languageselector">
```

Comment:
Bugfix?

Reply:
Yes Language selector fix to start the support of i18n in this application